roofit/hs3/test/testRooFitHS3.cxx

```diff
@@ -43,10 +48,59 @@ TEST(RooFitHS3, RooArgusBG)
    RooRealVar nbkg("nbkg", "#background events", 800, 0., 10000);
    RooAddPdf model("model", "g+a", RooArgList(signalModel, background), RooArgList(nsig, nbkg));
 
-   std::string rootetcPath = gSystem->Getenv("ROOTSYS");
-   RooJSONFactoryWSTool::loadExportKeys(rootetcPath + "/etc/root/RooFitHS3_wsexportkeys.json");
+   auto etcDir = std::string(TROOT::GetEtcDir());
+   RooJSONFactoryWSTool::loadExportKeys(etcDir + "/RooFitHS3_wsexportkeys.json");
+   RooJSONFactoryWSTool::loadFactoryExpressions(etcDir + "/RooFitHS3_wsfactoryexpressions.json");
+   
    RooWorkspace work;
    work.import(model);
    RooJSONFactoryWSTool tool(work);
    tool.exportJSON("argus.json");
 };
+
+TEST(RooFitHS3, SimultaneousGaussians)
+{
+   using namespace RooFit;
+
+   // Import keys and factory expressions files for the RooJSONFactoryWSTool.
+   auto etcDir = std::string(TROOT::GetEtcDir());
+   RooJSONFactoryWSTool::loadExportKeys(etcDir + "/RooFitHS3_wsexportkeys.json");
+   RooJSONFactoryWSTool::loadFactoryExpressions(etcDir + "/RooFitHS3_wsfactoryexpressions.json");   
+
+   // Create a test model: RooSimultaneous with Gaussian in one component, and
+   // product of two Gaussians in the other.
+   RooRealVar x("x", "x", -8, 8);
+   RooRealVar mean("mean", "mean", 0, -8, 8);
+   RooRealVar sigma("sigma", "sigma", 0.3, 0.1, 10);
+   RooGaussian g1("g1", "g1", x, mean, sigma);
+   RooGaussian g2("g2", "g2", x, mean, RooConst(0.3));
+   RooProdPdf model("model", "model", RooArgList{g1, g2});
+   RooGaussian model_ctl("model_ctl", "model_ctl", x, mean, sigma);
+   RooCategory sample("sample", "sample", {{"physics", 0}, {"control", 1}});
+   RooSimultaneous simPdf("simPdf", "simultaneous pdf", sample);
+   simPdf.addPdf(model, "physics");
+   simPdf.addPdf(model_ctl, "control");
+   
+   // this is a handy way of triggering the creation of a ModelConfig upon re-import
+   simPdf.setAttribute("toplevel");
+
+   // Export to JSON
+   {
+      RooWorkspace ws{"workspace"};
+      ws.import(simPdf);
+      RooJSONFactoryWSTool tool{ws};
+      tool.exportJSON("simPdf.json");
+      // Output can be pretty-printed with `python -m json.tool simPdf.json`
+   }
+
+   // Import JSON
+   {
+      RooWorkspace ws{"workspace"};
+      RooJSONFactoryWSTool tool{ws};
+      tool.importJSON("simPdf.json");
+     ws.Print();
```

Comment:
```suggestion
```
The print should be removed, the tests should not print anything if there is no error.